style.css

```diff
@@ -0,0 +1,701 @@
+/*
+black: #272a31
+orange: #ec5242
+silver: #d3d3d3
+font-family: cocogoose, lato */
+body {
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
+.flex {
+  display: flex;
+}
+
+/* Mobile Menu */
```

Comment:
- [ ] OPTIONAL
As requested by the previous code reviewer(s), it is a good practice to ensure that **all** comments are removed from the code to pushed to the repository. This will keep our code cleaner. Please let's endeavor to remove **all** comments found in our code, and rather give more meaningful names to the elements

Reply:
alright, I removed them all. thank you